libpdbg/device.c

```diff
@@ -701,6 +701,73 @@ static void pdbg_targets_init_virtual(struct pdbg_target *node, struct pdbg_targ
 	}
 }
 
+/**
+ * @brief It releases/deletes a target from its target_class list.
```

Comment:
it's

Reply:
I think here it's is not needed as I am putting a s with "delete" and "release" verbs